unit/Makefile

```diff
@@ -30,13 +25,42 @@ LIBS = ../src/ansi-c/ansi-c$(LIBEXT) \
        ../src/assembler/assembler$(LIBEXT) \
        ../src/analyses/analyses$(LIBEXT) \
        ../src/solvers/solvers$(LIBEXT) \
+       ../src/util/util$(LIBEXT) \
+       # Empty last line
+
+OBJ += ../src/ansi-c/ansi-c$(LIBEXT) \
+       ../src/cpp/cpp$(LIBEXT) \
+       ../src/json/json$(LIBEXT) \
+       ../src/linking/linking$(LIBEXT) \
+       ../src/util/util$(LIBEXT) \
+       ../src/big-int/big-int$(LIBEXT) \
+       ../src/goto-programs/goto-programs$(LIBEXT) \
+       ../src/pointer-analysis/pointer-analysis$(LIBEXT) \
+       ../src/langapi/langapi$(LIBEXT) \
+       ../src/assembler/assembler$(LIBEXT) \
+       ../src/analyses/analyses$(LIBEXT) \
+       ../src/solvers/solvers$(LIBEXT) \
+       # Empty last line
 
 CLEANFILES = $(SRC:.cpp=$(EXEEXT))
 
 all: $(SRC:.cpp=$(EXEEXT))
 
+catch: catch_entry_point$(EXEEXT)
+
+# Run the catch tests
+test-catch: catch
```

Comment:
This rule should not be named for the unit test framework. `unit-test` or `unit` might be better names.